content/donate.adoc

```diff
@@ -3,47 +3,79 @@ layout: simplepage
+** Cost of SaaS services not covered by sponsorships
+* Organizing link:https://www.jenkins.io/events/online-meetup/[Jenkins Online Meetups], link:https://www.jenkins.io/projects/jam/[local meetups] (swag, promotion materials)
+* Gifts for link:/security/#reporting-vulnerabilities[reporting security vulnerabilities] (link:https://www.jenkins.io/security/gift/[more info])
+* Funding the jep:12[Jenkins Travel Grant Program] to help community members to attend Jenkins related events.
 
-Your contribution is *not* used for paying personnel.
+Your contribution is *NOT* used for paying personnel, for placing adds, and for other forms of commercial promotion.
 
-== Ways to donate
+== Ways to donate money
 
 There are multiple ways to donate money to the Jenkins project.
 
-=== Donations through SPI
+=== Donations through Community Bridge
+
+image:/images/governance/funding/communitybridge.png[Donate via Community Bridge, link="https://funding.communitybridge.org/projects/jenkins", role=center, height=48]
 
-Before the transition to link:https://cd.foundation/[Continuous Delivery Foundation] in 2019,
-Jenkins project was under the umbrella of link:http://www.spi-inc.org/[Software in the Public Interest (SPI)].
-At the moment of this page update (Dec 26, 2019), SPI still accepts donations to the Jenkins project.
-In the future SPI will stop accepting donations.
+The Jenkins project https://funding.communitybridge.org/projects/jenkins[has an account] on the Community Bridge funding platform.
+It provides support for donations by individuals and by organizations, including anonymous donations.
+Minimum donation is 5USD for individuals and 500 USD for organizations.
 
-* You can do a link:https://co.clickandpledge.com/advanced/default.aspx?wid=46160[donation online] in USD
-* For other ways to donate, refer to the link:https://spi-inc.org/donations[donations page on the SPI website]
+* You can do a one-time donation or configure monthly donations using your credit card.
+* Organizations can also do a donation via invoice
 
-=== Donations through ffis.de
+See jep:15[Jenkins funding on Community Bridge] for more details about the donation process and targets.
 
-WARNING: Status of this donation way is not clear, see jira:WEBSITE-703[]
+=== Donations through Continuous Delivery Foundation 
 
-For European residents, we can also accept donations through link:http://www.ffis.de/Verein/donations.html[ffis.de] via bank transfer in EUR.
-As ffis.de is a charitable association, this donation is tax deductible in Germany.
+Large donations can be also done through link:https://cd.foundation/[Continuous Delivery Foundation].
+Please contact the link://jenkinsci-board@googlegroups.com[Jenkins Governance Board] if you want to make such a donation.
 
-NOTE: If you donate via ffis.de, please write to `donation[at]ffis.de` to ensure that your donation will be earmarked for the Jenkins project.
+=== Obsolete donation ways
```

Comment:
does this need to stay?

Reply:
Nope, I can remove it entirely if this is a preference

Reply:
I would remove, why does someone wanting to donate care about an old way if the new way works better